plugins/providers/virtualbox/cap/configure_disks.rb

```diff
@@ -108,6 +108,45 @@ def self.handle_configure_disk(machine, disk, all_disks)
           disk_metadata
         end
 
+        # Helper method to get the UUID of a specific controller attachment
+        #
+        # @param [Vagrant::Machine] machine - the current machine
+        # @param [String] controller_name - the name of the controller to examine
+        # @param [String] port - port to look up
+        # @param [String] device - device to look up
+        def self.attachment(machine, controller_name, port, device)
```

Comment:
This could probably be a driver method rather than a helper method here. And it could be called something like `get_disk_uuid` so it's a bit more descriptive to what it's doing. The method docs should also list the `# @return` value too!

Reply:
This was moved into the driver as part of the storage controller refactor.